server/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/RangeHistogramAggregatorTests.java

```diff
@@ -57,7 +57,7 @@ public void testDoubles() throws Exception {
                 new RangeFieldMapper.Range(rangeType, 1.0D, 5.0D, true, true), // bucket 0 5
                 new RangeFieldMapper.Range(rangeType, -3.1, 4.2, true, true), // bucket -5, 0
                 new RangeFieldMapper.Range(rangeType, 4.2, 13.3, true, true), // bucket 0, 5, 10
-                new RangeFieldMapper.Range(rangeType, 42.5, 49.3, true, true), // bucket 40, 45
+                new RangeFieldMapper.Range(rangeType, 22.5, 29.3, true, true), // bucket 40, 45
```

Comment:
let's either update the comment to match the new range, or just delete the comments.  And in the later tests as well.

Reply:
++, I pushed https://github.com/elastic/elasticsearch/pull/60683/commits/1dd98d16f924040dc870ab26c8feee62ad65e3ab